refinedstorage2-platform-fabric/build.gradle

```diff
@@ -68,7 +68,9 @@ dependencies {
     // We test with REI on Fabric.
     modRuntimeOnly fabric.bundles.rei.runtime
 
-    modCompileOnly fabric.bundles.jei
+    modCompileOnly fabric.jei.common.api
+    testCompileOnly fabric.jei.common.api
```

Comment:
not sure why, but if you do not add it the tests do not compile, since the need to compile stuff that uses the jei api.

You get an error in the compileTestClasses that
> error: package mezz.jei.api.helpers does not exist
in all classes that use IPlatformFluidHelper

Reply:
Yes works with implementation